src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,10 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball [] balls = new Ball[3];
+        for (int i = 0; i < 3; i++) {
+            balls[i] = new Lottery().getRandomBall();
```

Comment:
Don’t create several instances of the Lottery class if you can use only one instance for your purposes